documentation/library_guide/api_for_dpcpp_kernels/tested_standard_cpp_api.rst

```diff
@@ -358,6 +358,18 @@ C++ Standard API                     libstdc++  libc++     MSVC
 ``std::move``                        Tested     Tested     Tested
 ------------------------------------ ---------- ---------- ----------
 ``std::move_backward``               Tested     Tested     Tested
+------------------------------------ ---------- ---------- ----------
+``std::abs``                         Tested     Tested     Tested
```

Comment:
Claiming `abs` tested may need more tests, because there are overloads for it in `<cstdlib>` and `<cmath>`; please check if we already have tests for that.

Reply:
We have only two files with std::abs usages:
test\parallel_api\experimental\for_loop.pass.cpp
test\parallel_api\experimental\for_loop_induction.pass.cpp
So I think that now we don't have such tests.
Should I remove std::abs from list of tested functions ?

Reply:
Done

Reply:
Done